### ISSUE TITLE
EXLUDE_FILES ohne Wirkung

### ISSUE THREAD

**Issue by rrunner64** [linuzer/HardlinkBackup#1]
Kann es sein, dass die Funktion zum Ignorieren von Dateien und/oder Verzeichnissen in HardLink gar nicht implementiert ist?

**Comment by linuzer** [comment]
Richtig, die Funktion ist in [HardLink](https://github.com/linuzer/HardLink) (noch) nicht implementiert, aber ich freu mich über jeden PullRequest ;-)
